src/Forms/SearchableDropdownTrait.php

```diff
@@ -265,6 +265,13 @@ public function getSource(): array
         return $this->getListMap($this->sourceList);
     }
 
+    public function Field($properties = [])
+    {
+        $context = $this;
+        $this->extend('onBeforeRender', $context, $properties);
+        return $context->customise($properties)->renderWith($context->getTemplates());
+    }
```

Comment:
Took inspiration from tagfield here. This is really all we need and doing this means we avoid calling `getSource()` when trying to render the field.